pybop/plotting/plot_problem.py

```diff
@@ -3,9 +3,10 @@
 import numpy as np
 
 from pybop import DesignProblem, FittingProblem, StandardPlot
+from pybop.models.base_model import Inputs
 
 
-def quick_plot(problem, parameter_values=None, show=True, **layout_kwargs):
+def quick_plot(problem, inputs: Inputs = None, show=True, **layout_kwargs):
```

Comment:
I think in this situation, `parameter_values: Inputs` provides more clarity to the user. Plotting `inputs` could be quite a few things.

Reply:
The phrase `parameter_values` is not ideal as the user should pass an Inputs dictionary here. Perhaps `problem_inputs` would work?

Reply:
`problem_inputs` works for me :+1: